src/Button/SimpleButton/SimpleButton.less

```diff
@@ -19,4 +19,36 @@ button.react-geo-simplebutton {
     background-color: @component-background;
     border-color: darken(@component-background, 10);
   }
+
+  @media screen and (max-width: 1280px) {
```

Comment:
The threshold value of `1280px` could possibly be moved to a less variable (see [here](https://github.com/terrestris/react-geo/blob/master/src/style/variables.less)).